modules/rubiconAnalyticsAdapter.js

```diff
@@ -363,8 +363,8 @@ export function parseBidResponse(bid, previousBidResponse, auctionFloorData) {
       const height = bid.height || bid.playerHeight;
       return (width && height) ? {width, height} : undefined;
     },
-    'pbsBidId',
-    'seatBidId',
+    'pbsBidId', pbsBidId => pbsBidId == 0 ? utils.generateUUID() : pbsBidId,
+    'seatBidId', seatBidId => seatBidId == 0 ? utils.generateUUID() : seatBidId,
```

Comment:
So this logic would be as follows:

If set and is 0 or '0'

then overwrite it

else pass it along

If it is undefined (most often case) it will be ignored downstream and not set.